cmd/api/main.go

```diff
@@ -1,16 +1,37 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
 	"log"
 	"net/http"
+	"os"
 
 	"github.com/wikidistance/wikidist/pkg/api"
 	"github.com/wikidistance/wikidist/pkg/db"
 )
 
 func main() {
-	d, err := db.NewDGraph()
+	// Get filename from args
+	filename := "../../config.json"
+	if len(os.Args) == 2 {
+		filename = os.Args[1]
+	}
+
+	// Get config from file
+	file, err := os.Open(filename)
```

Comment:
nit: we can be a slightly more straightforward by using `ioutil.ReadFile` and `json.Unmarshal`